R/missingness.R

```diff
@@ -134,3 +135,41 @@ countMissingData <- function(x, userNAs = NULL) {
   .Deprecated("missingness", "healthcareai")
   missingness(x)
 }
+
+
+#' Replaces missing data with NA
+#'
+#' This function replaces any given value in a dataframe with NA.
+#'
+#' @param d A dataframe or matrix
+#' @param to_replace The value that will be replaced.
+#' @return A dataframe with the missing data replaced.
+#'
+#' @export
+#' @example
+#' # Replace "NULL" in dat
+#' dat <- data.frame(a = c(1, 2, "NA", NA, "none", "??", "?", 5),
+#'                   b = c("blank", 0, "na", "None", "none", 3, 10, 4),
+#'                   c = c(10, 5, 8, 1, NA, "NULL", NaN, "Nas"),
+#'                   d = c(1, 6, 7, 8, 9, 5, 10, 1078950492))
+#'
+#' replace.missingness(dat, "NULL")
+#'
+replace.missingness <- function(d, to_replace) {
+  if (is.data.frame(d)) {
+    d[d == to_replace] <- NA
+
+    d <- map_dfr(d, function(col) {
+      if (is.character(col)) {
+        ans <- sum(is.na(col)) == sum(is.na(as.numeric(col)))
+        if (ans) {
+          col <- as.numeric(col)
```

Comment:
Can you add a suppress warnings in those `as.numeric` conversions so that we don't get them spewed to the console?